DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -247,6 +259,14 @@ public override void Initialize()
             PostUIInit();
         }
 
+        private void OnCnCNetGameCountUpdated(object sender, PlayerCountEventArgs e)
+        {
+            lock (locker)
+            {
+                lblOnline.Text = "Online: " + e.PlayerCount.ToString();
+            }
```

Comment:
Locking might not be necessary here, although MainMenu does it too. I'll check later if it's really needed.